lib/modules/manager/helmv3/index.ts

```diff
@@ -14,4 +15,6 @@ export const defaultConfig = {
   fileMatch: ['(^|/)Chart\\.yaml$'],
 };
 
+export const categories: Category[] = ['helm'];
```

Comment:
How about `kubernetes` too?

Reply:
🤔  
In that case we would have to add to all `helm` managers, which would add 5 entries to the `kubernetes` group.

Reply:
What if we remove `helm` as a category and put it with `kubernetes`?

Reply:
We have 5 managers which gravitate around helm therefore, I think this is enough to facilitate its own category.  Further Kubernetes would be then quite big with 10 managers. 

What do you think about adding references to other categories e.g. `helm` implies the `kubernetes` category and this would be represented like this in the manager overview:


**kubernetes**: [`argocd`](argocd/), [`flux`](flux/), [`jsonnet-bundler`](jsonnet-bundler/), [`kubernetes`](kubernetes/), [`kustomize`](kustomize/), [`category:helm`]() 


But I think this should be added in another PR following up to this one.

Reply:
I'd prefer not to have inheritance. Would be better for them to simply have both

Reply:
K, in that case I will add the `kubernetes` category to the ones with `helm`.